### ISSUE TITLE
Gebruikersprofiel maken

### ISSUE THREAD

**Issue by jtborger** [deregenboog/ecd#1362]
Het zou mooi zijn een gebruikersprofiel te maken waarin bepaalde instellingen per gebruiker opgeslagen kunnen worden.
Zoals bv de gewenste startpagina, evt icm welk filter ingeladen moet zijn.
Maar bv ook lettergrootte. En evt andere nog te bedenken opties?

**Comment by barthuttinga** [comment]
@jtborger Is hier door gebruikers om gevraagd? Maakt het misschien onnodig complex.

Op basis van rol naar een logische startpagina gaan zou nog wel een mooie optie kunnen zijn (als je rol bepaalt dat je eigenlijk maar een module mag zien, heeft het weinig zin om eerst naar het "dashboard" te gaan waar dan vervolgens maar een linkje staat).

**Comment by jtborger** [comment]
Nee, niet vanuit gebruikers is een eigen idee om het wat moderner te maken. Wat ik wel merk is dat sinds 2 jaar de organisatie is verdubbeld en dat nieuwe mensen die van andere organisaties komen, een moderner IT systeem gewend zijn... Ik zoek naar laaghangend fruit in ECD om het werken efficiënter te maken voor ze, en ook wat moderner. Zoals bv een eigen startpagina, of het kunnen opslaan van een filter zodat je als het ware je eigen 'Mijn <dashboard>' zou kunnen maken. 
Maar het is dus geen concrete gebruikersvraag.

Het rollen verhaal is idd wel een goed idee en makkelijker te realiseren.
Als je er geen gevoel of idee bij hebt laten we dit nog even liggen.
